stable/graphite/templates/statefulset.yaml

```diff
@@ -8,25 +8,19 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  strategy:
-    type: Recreate
+  updateStrategy:
+    type: RollingUpdate
   selector:
     matchLabels:
       app: {{ template "graphite.name" . }}
```

Comment:
As we're implement braking changes anyway it woul be a good opporunity to change all labes to the pattern helm > 2.11. uses...

Reply:
I think this should be merged as is and a second PR can follow that. The chart as it is is broken - it doesn't work.